Samples/README.md

```diff
@@ -35,13 +69,49 @@ PWMapKit Samples for iOS
 - Specify a `poiLocation` that is within the bounds of your building
 - Optionally fill out configuration values in `addCustomPointOfInterest` such as `poiFloorId`, `poiTitle` etc.
 
+##### Sample code in [CustomPOIViewController.swift](https://github.com/phunware/maas-mapping-ios-sdk/blob/readme/Samples/MapScenarios/MapScenarios/Scenarios/CustomPOIViewController.swift):
+
+```
+func addCustomPointOfInterest() {
+	let poiLocation = CLLocationCoordinate2DMake(30.359931, -97.742507)
+	let poiTitle = "Custom POI"
+        
+	// If the image parameter is nil, it will use the POI icon for any specified `pointOfInterestType`. If no image is set and no `pointOfInterestType` is set, the SDK will use this default icon: https://lbs-prod.s3.amazonaws.com/stock_assets/icons/0_higher.png
+	let customPOI = PWCustomPointOfInterest(coordinate: poiLocation, floor: mapView.currentFloor, title: poiTitle, image: nil)
+        
+	customPOI?.isShowTextLabel = true
+        
+	if let customPOI = customPOI {
+		mapView.addAnnotation(customPOI)
+	}
+}
+```
+
 ### LocationModesViewController
 - Button to switch the map's camera mode to follow current location, not follow current location, or follow current location and align the camera to the device compass
 
 ##### Usage:
 - Fill out `applicationId`, `accessKey`, `signatureKey`, and `buildingIdentifier`
 - Tap the arrow button to visualize the various location modes
 
+##### Sample code in [LocationModesViewController.swift](https://github.com/phunware/maas-mapping-ios-sdk/blob/readme/Samples/MapScenarios/MapScenarios/Scenarios/LocationModesViewController.swift):
+
+```
+extension LocationModesViewController: PWMapViewDelegate {
+    
+    func mapView(_ mapView: PWMapView!, didChangeIndoorUserTrackingMode mode: PWTrackingMode) {
+        switch mode {
+        case .none:
+            trackingModeButton.image = // .emptyTrackingImage(color: .blue)
+        case .follow:
+            trackingModeButton.image = // .filledTrackingImage(color: .blue)
+        case .followWithHeading:
+            trackingModeButton.image = // .trackWithHeadingImage(color: .blue)
```

Comment:
can we show the function that changes the PWMapView's tracking mode instead? this is just showcasing a UI outside the SDK